src/App/Hovedside/TjenesteBoksContainer/Arbeidstreningboks/Arbeidstreningboks.tsx

```diff
@@ -1,17 +1,73 @@
-import React, { FunctionComponent } from "react";
+import React, {
+  FunctionComponent,
+  useContext,
+  useEffect,
+  useState
+} from "react";
 import arbeidstreningikon from "./arbeidstreningikon.svg";
 import Lenkepanel from "nav-frontend-lenkepanel";
 import "./Arbeidstreningboks.less";
 
 import TjenesteBoksBanner from "../TjenesteBoksBanner/TjenesteBoksBanner";
 import { syfoLink } from "../../../../lenker";
+import { OrganisasjonsDetaljerContext } from "../../../../OrganisasjonDetaljerProvider";
+import { Arbeidsavtale } from "../../../../api/dnaApi";
 
 interface Props {
   varseltekst?: string;
   className?: string;
 }
 
 const Arbeidstreningboks: FunctionComponent<Props> = props => {
+  const { arbeidsavtaler } = useContext(OrganisasjonsDetaljerContext);
+  const [
+    antallKlareStillingsannonserTekst,
+    setantallKlareStillingsannonserTekst
+  ] = useState("");
+  const [antallTilGodkjenningTekst, setantallTilGodkjenningTekst] = useState(
+    ""
+  );
+  const [antallUnderArbeidTekst, setantallUnderArbeidTekst] = useState("");
+
+  const LagTekstBasertPaAntall = (antall: string) => {
+    if (antall === "1") {
+      return " arbeidsavtale ";
+    }
+    return " arbeidsavtaler ";
+  };
+
+  useEffect(() => {
+    const KlareForOppstartArbeidsavtaler: Array<
+      Arbeidsavtale
+    > = arbeidsavtaler.filter(
+      arbeidsavtale => arbeidsavtale.status === "Klar for oppstart"
+    );
+    let antallAvtaler: string = KlareForOppstartArbeidsavtaler.length.toString();
```

Comment:
Hva med å ikke gjenbruke variabelen `antallAvtaler` her, men heller lage flere variabler som heter `antallKlareAvtaler`, `antallGodkjenteAvtaler` osv? Det kan gjøre det lettere å lese. Generelt prøver vi å bruke `const` i stedet for `let`, så hvis man bruker `let` betyr det ofte at man kan gjøre ting på en litt annen måte.